server/views/pages/contacts/manage/contactDetails/manageDomesticStatus.njk

```diff
@@ -0,0 +1,59 @@
+{% extends "../../../../partials/layout.njk" %}
+{% from "govuk/components/input/macro.njk" import govukInput %}
+{% from "govuk/components/select/macro.njk" import govukSelect %}
+{% from "govuk/components/button/macro.njk" import govukButton %}
+{% from "../../../../partials/miniProfile/macro.njk" import miniProfile %}
+{% from "govuk/components/back-link/macro.njk" import govukBackLink %}
+
+{% set pageTitle = applicationName + " - Enter domestic status" %}
+{% set title = "What is the domestic status of " + contact | reverseFormatName + "?" %}
+
+{% block content %}
+
+{{ govukBackLink({
+    text: "Back",
+    href: '/prisoner/' + prisonerDetails.prisonerNumber + '/contacts/manage/' + contact.id,
+    attributes: {"data-qa": "back-link"}
+}) }}
+
+{{ miniProfile(prisonerDetails) }}
+<div class="govuk-grid-row">
+    <div class="govuk-grid-column-one-half">
+        <span class="govuk-caption-l">Contacts</span>
+        <h1 class="govuk-heading-l" data-qa="main-heading">{{ title }}</h1>
+        <form method='POST'>
+             <input type="hidden" name="_csrf" value="{{ csrfToken }}" />
+             {% set domesticStatusOptions = [{
+                value: '',
+                text: 'Select',
```

Comment:
elsewhere the blank option is blank text. I think we want that here as well. Also if you look at title, relationship, etc. the options are generated in the controller because here you have both blank and the previous option `selected`.

Reply:
Options generated in the controller